README.md

```diff
@@ -31,6 +31,13 @@ sh run_analysis.sh
 
 ## Contribution Notes
 
-- We use the [one sentence per line](https://nick.groenen.me/notes/one-sentence-per-line/) convention for markdown files.
-- We use [ruff](https://github.com/astral-sh/ruff) for linting and formatting code.
-- We use the [Google format](format) for function docstrings.
+We use [ruff](https://github.com/astral-sh/ruff) for linting and formatting code.
+
+We use the following conventions:
+- [One sentence per line](https://nick.groenen.me/notes/one-sentence-per-line/) convention for markdown files
+- [Google format](format) for function docstrings
+- [tsv](https://en.wikipedia.org/wiki/Tab-separated_values#:~:text=Tab%2Dseparated%20values%20(TSV),similar%20to%20comma%2Dseparated%20values.) file format for saving small dataframes that require easy readability
+- [hdf5](https://www.hdfgroup.org/solutions/hdf5/) file format for saving large dataframes
+- Data location information (well, tile, cycle, etc) + `__` + type of information (cell features, phenotype info, etc) + `.` + file type. 
+Data is stored in its respective analysis directories. 
+For example: `analysis_root/preprocess/ic_fields/sbs/A1_T1_C1__ic_field.tiff`
```

Comment:
The file naming convention is great. Is the analysis directory fixed in this way?

Reply:
That is an actual path yes. I'm open to more discussion about the organization of processing directories. In this case it made sense to me to have module (preprocess) -> subprocess (ic_fields) -> data type (sbs or ph; this is only a relevant distinction for preprocessing module). 

Basically with this file convention I was saying that the file names consistently contain:
1) Data location (well, tile, cycle)
2) Information type (ic_field, aligned_image, etc)
3) File type (tiff, tsv, etc)
And all other information (module, subprocess, sbs vs ph) gets stored as directories. Let me know if my logic is off here.